xchange-okex/src/main/java/org/knowm/xchange/okex/v5/service/OkexMarketDataServiceRaw.java

```diff
@@ -51,4 +54,24 @@ public OkexMarketDataServiceRaw(
       throw handleError(e);
     }
   }
+
+  public OkexResponse<List<OkexTrade>> getOkexTrades(String instrument)
+      throws OkexException, IOException {
+    try {
+      return okex.getTrades(instrument, 30);
+    } catch (OkexException e) {
+      throw new OkexException(e.getMessage(), e.getHttpStatusCode());
+    }
```

Comment:
doing this look pointless, just let the original OkexException exception escape the method

Reply:
indeed!

Reply:
removed